scripts/dashboard-importer/import.sh

```diff
@@ -14,36 +14,34 @@
 # limitations under the License.
 
 JSON_PATH=$1 # Path to directory containing JSON files or a JSON file
-PROJECT=$2 # Supplied GCP project to upload to
+PROJECT=$2 # Optionally supplied GCP project to upload to
 
 # Validate Arguments
 if [ "$JSON_PATH" = "" ]
 then
   echo "Error: No JSON_PATH included"
-  echo "Syntax: import.sh <JSON_PATH> <PROJECT>"
-  exit
-fi
-
-if [ "$PROJECT" = "" ]
-then
-  echo "Error: No PROJECT included"
-  echo "Syntax: import.sh <JSON_PATH> <PROJECT>"
+  echo "Syntax: ./import.sh <JSON_PATH> <PROJECT>"
```

Comment:
Can this echo reflect that PROJECT is optional?

Reply:
Added  OPTIONAL_ in front of PROJECT, not sure if there's a better way, but open to suggestions!